src/ReactHTMLElement.ts

```diff
@@ -35,11 +35,10 @@ class ReactHTMLElement extends HTMLElement {
     this._mountPoint = mount;
   }
 
-  render(
-    app: Parameters<ReactDOM.Renderer>[0][number]
-  ): ReturnType<ReactDOM.Renderer> {
-    // eslint-disable-next-line react/no-render-return-value
-    return ReactDOM.render(app, this.mountPoint);
+  render(app: Parameters<ReactDOM.Renderer>[0][number]): void {
+    if (!this.isConnected) return;
```

Comment:
We make this check downstream quite often. It makes sense to me to do this here so rendering is Always Safe™.